apps/server/src/modules/school/service/school.service.ts

```diff
@@ -31,13 +30,13 @@ export class SchoolService {
 		oauthMigrationFinished?: boolean
 	): Promise<OauthMigrationDto> {
 		const schoolDo: SchoolDO = await this.schoolRepo.findById(schoolId);
-		if (isDefined(oauthMigrationPossible)) {
+		if (oauthMigrationPossible != null) {
 			schoolDo.oauthMigrationPossible = oauthMigrationPossible ? new Date() : undefined;
 		}
```

Comment:
Why not? 
schoolDo.oauthMigrationPossible = oauthMigrationPossible === true ? new Date() : undefined;

Then you don't need if statement